Framework/Core/include/Framework/AnalysisDataModel.h

```diff
@@ -665,6 +665,22 @@ DECLARE_SOA_TABLE(CaloTriggers, "AOD", "CALOTRIGGER", //! Trigger information fr
                   calotrigger::LnAmplitude, calotrigger::TriggerBits, calotrigger::CaloType);
 using CaloTrigger = CaloTriggers::iterator;
 
+namespace cpvcluster
+{
+DECLARE_SOA_INDEX_COLUMN(BC, bc);                                  //! BC index
+DECLARE_SOA_COLUMN(PosX, posX, float);                             //! X position in cm
+DECLARE_SOA_COLUMN(PosZ, posZ, float);                             //! Z position in cm
+DECLARE_SOA_COLUMN(Amplitude, amplitude, float);                   //! Signal amplitude
+DECLARE_SOA_COLUMN(PadMult, padMult, uint8_t);                     //! Multiplicity of pads in cluster
+DECLARE_SOA_COLUMN(ModuleNumber, moduleNumber, uint8_t);           //! CPV module number (2, 3 or 4)
+DECLARE_SOA_COLUMN(LocalMaximaNumber, localMaximaNumber, uint8_t); //! Number of local maxima in cluster
```

Comment:
What is the range of this value? Can we combine the previous two into one value and then use dynamical columns to expand them?

Reply:
Sorry, I didn't understand the question? Just to clarify: Amplitude is total cluster amplitude, PadMult is number of pads in cluster, ModuleNumber is CPV module number in which the cluster is found, LocalMaximaNumber is number of local maxima in the cluster. I don't think you can combine any of this value out of the rest. Let's say the variables construct orthogonal basis in space of cpv clusters.

Reply:
The comment was on LocalMaximaNumber and ModuleNumber
ModuleNumber needs 2 bits
How many bits are needed by LocalMaximaNumber?

Reply:
I.e. what is the highest expected number of local maxima?

Reply:
Ah, I see what you mean. I'll check the highest expected number of local maxima with @peressounko 
Don't think it is too high.

Reply:
So I checked and found that number of local maxima is fictional name. It is 1 if cluster is unfolded and is 2 if cluster is 
not unfolded. So actually it is only one bit. And PadMult, ModuleNumber and LocalMaximaNumber can be actually packed into 8 bits. Thanks for digging in this direction. I'll modify the code accordingly.

Reply:
Great! You can then add dynamic columns (look for examples in the same file) to make related analysis code more readable.